mycore-base/src/main/java/org/mycore/common/xml/MCRXMLFunctions.java

```diff
@@ -659,6 +659,21 @@ public static String getDisplayName(String classificationId, String categoryId)
     }
 
     public static boolean isCategoryID(String classificationId, String categoryId) {
+        if (classificationId == null || classificationId.isEmpty()) {
+            LOGGER.debug("Classification identifier is null or empty");
+            return false;
+        }
+
+        if (categoryId == null || categoryId.isEmpty()) {
+            LOGGER.debug("Category identifier is null or empty");
+            return false;
+        }
+
+        if (classificationId.length() > 32) {
```

Comment:
a comment, explaining the reason for the `32`, might be helpful for future readers

Reply:
changed '32' to MCRCategoryID.ROOT_ID_LENGTH